element-hq/element-x-android
# Don't delete uploaded logs

## Content

Remove code in `DefaultBugReporter` that deleted all existing log files except for the most recent one. 

## Motivation and context

The log removal happened after the logs had been compressed but before we were sure they had reached their destination. Since Rust already takes care of the log roration there's no need for this code AFAICT, and it could cause accidental log data loss.

## Tests



- Send a bug report.
- Send another or check the files in your device with `adb`.

The log files should be kept.

## Tested devices

- OS version(s): 14

## Checklist
